src/content/changes/highlights/2021-07-16.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2021-07-16T10:00:00
+title: Highlights for the week ending July 16, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Managed apps
+
+[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Breaking**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable, view [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) for details.
```

Comment:
```suggestion
[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Breaking change**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable it please check the details in the [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/).
```

Reply:
maybe we can explain it a bit more in the announcement. I do think none of our customers use `ExternalName` with Ingress Controller but just in case I will add a bit more of context.

Reply:
yes please. Maybe write it more scary than it is to nudge towards upgrading?